### PR TITLE
Added error handling to earthquake_api

### PR DESCRIPTION
**Pull request by MAvecillas23**
Added error handling for earthquake_api modules earthquake_main function and earthquake_data_request function.

earthquake_main(): added if statements and returns. If there was an error with the earthquake api request, return that message to where it was called... (Flask's app.py) If there were no errors with the request then call the location_magnitude_date() function and get needed data from the earthquake object and then return that list of data back to where it was called (Flasks app.py).

earthquake_data_request() : Added a generic try except block... let me know if there are any specific request errors you need me to add.

main.py main function: Added "return coordinates" to be able to get coordinates to run for earthquake.py

### CODE REVIEW

File: earthquake_api.py [hunk 1]
```diff
@@ -47,10 +59,12 @@ def location_magnitude_date(data):
 
     return magnitude_location_date_list
 
+
 # displays the location, magnitude, data list data to the user
+# **** This function is used for testing that earthquake data is being returned ***
 def display_earthquake_data(needed_data):
```
Comment (littlestripes) [inline earthquake_api.py:66]: I think this could be one of the unit tests for this API.

Comment (Solis13) [reply earthquake_api.py:66]: This looks great, good to go for merging

File: earthquake_api.py [hunk 2]
```diff
@@ -19,16 +18,29 @@
...
+        return get_wanted_data
 
 # api call request
 # returns all earthquake data
 def earthquake_data_request(url):
-    earthquake_request = requests.get(url).json()
-    return earthquake_request
+    try:
+        earthquake_request = requests.get(url).json()
+        return earthquake_request, False  # return false if no errors occurred
```
[the remark this answers is not in the document]
Comment (Solis13) [reply earthquake_api.py:39]: I’d recommend removing the try-except block

File: earthquake_api.py [hunk 3]
```diff
@@ -59,4 +73,4 @@ def display_earthquake_data(needed_data):
 
 
 if __name__ == '__main__':
-    main()
\ No newline at end of file
+    earthquake_main()
```
[the remark this answers is not in the document]
Comment (MAvecillas23) [reply earthquake_api.py]: I think it would be easier if we just used the len() function on app.py

**Comment by MAvecillas23** [comment]
had to change up some outdated code, replaced the try except blocks with raise_for_status() in earthquake_data_request() function. See latest commit.

File: main.py [hunk 4]
```diff
@@ -5,6 +5,7 @@ def main():
     location = input('Please enter location you would like data on (city, state): ')
     coordinates = gc.getCoordinates(location)
     print(coordinates)  # testing
+    return coordinates
```
[the remark this answers is not in the document]
Comment (Solis13) [reply main.py:8]: I completely agree with that

**Review by Solis13 — COMMENTED** [review]
Everything looks good

**Comment by MAvecillas23** [comment]
Talked about this PR on zoom, looked good merging now.

**Pull request merged**